go/parquet/file/column_writer.go

```diff
@@ -179,7 +179,7 @@ func (w *columnWriter) TotalBytesWritten() int64 {
 }
 
 func (w *columnWriter) RowsWritten() int {
-	return w.rowsWritten
+	return w.rowsWritten + w.numBufferedRows
```

Comment:
not sure if this is needs a doc update or not?

Reply:
the documentation is still correct, I had to add that because of the way I split it up to ensure proper writing of rows vs values in the data pages themselves. `RowsWritten` is still the number of rows written to the column (but not necessarily flushed yet).